### ISSUE TITLE
Target

### ISSUE THREAD

**Issue by diegobot1977** [simple-icons/simple-icons#5893]
**Brand Name:** Target
**Website:** https://www.target.com/
**Alexa rank:** 255
**Official resources for icon and color:** Also retrieve _this color_ - `#cb0000`. See Below in the Target homepage.

**Comment by service-paradis** [comment]
@diegobot1977 There is no need to close already opened issues to open another one. Why did you close #5857?

**Comment by diegobot1977** [comment]
because is still waiting for requesting the Target icon

**Comment by service-paradis** [comment]
Yes, but there is no need to close your issue to open another one. It can take some time before someone work on them. Just keep this one open until someone want top make it.

**Comment by diegobot1977** [comment]
Ok.

**Comment by diegobot1977** [comment]
Ok. @service-paradis, consider previewing this one:

![target](https://user-images.githubusercontent.com/73037261/121402790-4b0dc300-c928-11eb-9902-83ce0877a93d.png)

I go with this Target icon, but not a logo with bullseye, and just wrote in SVG-edit. So, it should go on https://corporate.target.com/corporate-responsibility/marketing-resources.

**Comment by diegobot1977** [comment]
Ok, consider previewing this one:

![target](https://user-images.githubusercontent.com/73037261/121402790-4b0dc300-c928-11eb-9902-83ce0877a93d.png)

I go with this Target icon, but not a logo, and just wrote in SVG-edit. So, it should go on https://corporate.target.com/corporate-responsibility/marketing-resources.

**Comment by ericcornelissen** [comment]
Closing this in favor of #5857. @diegobot1977, please open issues for icon requests only once in the future.
